main.py

```diff
@@ -0,0 +1,64 @@
+import discord
+import os
+from discord.ext import commands
+from secrettoken import token
+from discord.ext.commands import has_permissions, MissingPermissions
+from discord.utils import get
+
+intents = discord.Intents.default()
+intents.presences = True
+intents.members = True
+ 
+client = commands.Bot(command_prefix='$', intents=intents)
+
+@client.command()
+async def load(ctx, extension):
```

Comment:
What are the purposes of the load/unload function? Are these going to be used in the future by other extensions? We're still calling client.load_extension on line 24.

Reply:
it is to unload/load the each individual cog for example like one that does a command. It's for testing also like for example if I wanted to do $move @LinsaneFC after unloading the cog then it won't work

Reply:
*unloading the cog move.py

Reply:
you also mention in the settings issue that we can "Settings for the bot should change the state of the bot and these commands should only be accessed by people with moderator accessibility or up for a particular server." Unloading and loading could be used to change the state of the bot so that we can disable commands and stuff like that.

Reply:
idk if that's what u meant by changing the state of the bot tho

Reply:
So these commands can be changed at runtime? As in I wouldn't have to rerun the bot or change any code and I can load/unload at runtime while the bot is running to change the state of the bot and disable/enable commands?

Reply:
hopefully yea in the discord general text channel.